host-configs/docker/docker-linux-ubuntu16.04-ivybridge-gcc@8.1.0.cmake

```diff
@@ -1,57 +1,44 @@
 #------------------------------------------------------------------------------
 # !!!! This is a generated file, edit at own risk !!!!
 #------------------------------------------------------------------------------
-# SYS_TYPE: linux-ubuntu16.04-ivybridge
-# Compiler Spec: gcc@8.1.0
-#------------------------------------------------------------------------------
 # CMake executable path: /usr/bin/cmake
 #------------------------------------------------------------------------------
 
 #------------------------------------------------------------------------------
 # Compilers
 #------------------------------------------------------------------------------
+# Compiler Spec: gcc@8.1.0
+#------------------------------------------------------------------------------
+if(DEFINED ENV{SPACK_CC})
 
-set(CMAKE_C_COMPILER "/usr/bin/gcc" CACHE PATH "")
+  set(CMAKE_C_COMPILER "/home/axom/axom_tpls/spack/lib/spack/env/gcc/gcc" CACHE PATH "")
```

Comment:
This is really the only fundamental different in the new host-configs generated by the new spack package.

This detects if axom is being built by inside of Spack so it uses their compiler wrappers.  The else block will continue our regular workflow of using the regular system compilers when not in spack.